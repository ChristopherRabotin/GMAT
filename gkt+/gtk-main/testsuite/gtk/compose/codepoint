<Multi_key> <U73> <U6F> <U7a> : "!"
