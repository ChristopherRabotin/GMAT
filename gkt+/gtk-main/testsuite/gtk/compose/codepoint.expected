# n_sequences: 1
# max_seq_len: 4
# n_index_size: 1
# data_size: 9
# n_chars: 0
<Uff20> <U73> <U6f> <U7a> : "!" # U21
